Assets/RedBlueGames/BulkRename/Editor/BulkRenamerWindow.cs

```diff
@@ -211,64 +210,33 @@ private void OnGUI()
             EditorGUILayout.EndScrollView();
         }
 
-        private string[] GetNamesFromSelections()
+        private void SortObjectsToRename()
         {
-
-            List<string> allAssetNames = new List<string>();
-            List<Object> allSortedGameObectsNames = new List<Object>();
-
-            int namesCount = this.objectsToRename.Count;
-            var names = new string[namesCount];
-            bool isItAsset = AssetDatabase.GetAssetPath(objectsToRename[0]) != "";
```

Comment:
So I couldn't figure out why you needed to parse an Asset path one way and a game object another. Object.name for a both game objects and assets returns just the asset name. I ended up removing it and just sorting them all the same way, according to name.length then name.

Another note - this had a bug where you couldn't rename both assets and gameobjects since it only queried the first object to figure out if it was an asset, then parsed all objects in the list. But since I removed it in favor of sorting all assets by name, it was no longer an issue for me.